apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -116,7 +116,7 @@ const TeamOutput: React.FC<TeamOutputProps> = ({
           permissions={permissions}
           onSave={(output) =>
             researchOutputData
-              ? updateResearchOutput(researchOutputData.id, {
+              ? updateAndPublishResearchOutput(researchOutputData.id, {
```

Comment:
This renaming seems weird. We were always doing a publish but the change is that we reset the recoil cache